Client/Frontend/Home/TopSites/DataManagement/TopSiteHistoryManager.swift

```diff
@@ -52,10 +34,7 @@ class TopSiteHistoryManager: DataObserver {
     }
 
     func removeTopSite(site: Site) {
-        profile.history.removeFromPinnedTopSites(site).uponQueue(dataQueue) { [weak self] result in
```

Comment:
Since we removed dataQueue usage here, we can remove that queue entirely I think (not used anymore)